SPECS/libpng/libpng.spec

```diff
@@ -23,7 +22,6 @@ It contains the libraries and header files to create applications
 
 %prep
 %setup -q
```

Comment:
Consider changing `%setup` to an `%autosetup` macro to better automate the patch application process
https://rpm-software-management.github.io/rpm/manual/autosetup.html